cobra/cmd/project_test.go

```diff
@@ -1,24 +1,20 @@
 package cmd
 
-import (
-	"testing"
-)
-
-func TestFindExistingPackage(t *testing.T) {
-	path := findPackage("github.com/OneCloudInc/cobra")
-	if path == "" {
-		t.Fatal("findPackage didn't find the existing package")
-	}
-	if !hasGoPathPrefix(path) {
-		t.Fatalf("%q is not in GOPATH, but must be", path)
-	}
-}
-
-func hasGoPathPrefix(path string) bool {
-	for _, srcPath := range srcPaths {
-		if filepathHasPrefix(path, srcPath) {
-			return true
-		}
-	}
-	return false
-}
+// func TestFindExistingPackage(t *testing.T) {
```

Comment:
do we know why this worked before (or did it?) and doesn't work now?

Reply:
it didn't. It was failing on both master and develop. OC just left it that way, but I'd like Skynet to run the unit tests.

Since this was the only one, I figured I'd just comment it out. It doesn't work because it's pre-Go Modules: it requires that the source code be located at `$GOPATH/src` in order to pass. However, Go Modules did away with all of that, and that's probably when the test started failing.

I even went out to the original repo and tried to see if they had an updated version of this test that would pass, but the repo has changed so much that I wasn't able to find anything meaningful.

Reply:
👍 that explanation works for me!